### ISSUE TITLE
Advanced Selectors: Add note not to do the test Skills in linked MDN article

### ISSUE THREAD

**Issue by danieletie** [TheOdinProject/curriculum#29241]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### Describe your suggestion

Referring to this [lesson](https://www.theodinproject.com/lessons/node-path-intermediate-html-and-css-advanced-selectorsl), there's a link to an article on the MDN and at the bottom of the article is a _"Test your skills!"_ section. 

<img width="353" alt="image" src="https://github.com/user-attachments/assets/69880a40-6340-4cb2-96f4-67dee0cc5994" />

Is it possible to have a note that says not to do the test your skills? This is because at this point in the lesson, there are skills tested that haven't been discussed. 

Maybe later on during the lesson, probably towards the end, a link [like so](https://developer.mozilla.org/en-US/docs/Learn_web_development/Core/Styling_basics/Combinators#test_your_skills!) to the Tesk Skills section of the article can be added which then tests knowledge all of the topics discussed during the lesson. This is because the skills tested are good and representetative of what is discussed in the lesson

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/node-path-intermediate-html-and-css-advanced-selectors

### (Optional) Discord Name

daniel.scrypt

**Comment by rlmoser99** [comment]
Thank you for taking the time to make this issue. I believe this link would be better to be moved to the Assignment section of the lesson. We have began to move most of the links in our lesson to be during an assignment, versus sprinkled throughout the lesson.

Do you think moving that link to the Assignment section would work better?

**Comment by danieletie** [comment]
Yes please. Thanks for responding. I think moving the link to the assignment section would work better.

**Comment by rlmoser99** [comment]
Sounds like a good plan. I've assigned you to this issue. When you make a PR, please reference this issue in the body of the PR.
